azure-servicebus/src/test/java/com/microsoft/azure/servicebus/primitives/TransportTypeTest.java

```diff
@@ -0,0 +1,114 @@
+package com.microsoft.azure.servicebus.primitives;
+
+import com.microsoft.azure.servicebus.TestUtils;
+import com.microsoft.azure.servicebus.amqp.ConnectionHandler;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+
+public class TransportTypeTest {
```

Comment:
All these tests access private fields in classes using reflection. They will break if the filed names are changed by an unsuspecting dev. These are too fragile and they are not adding any value. I would remove them.

Reply:
@yvgopal Based on the email discussion we had with @nemakam , it seems both TransportType and ProxySelectorTests were required as it tests what port and connection are handled.
@nemakam should both the tests be removed?